packages/suite/src/types/wallet/coinjoin.ts

```diff
@@ -31,7 +31,7 @@ export interface CoinjoinSession extends CoinjoinSessionParameters {
     timeCreated: number; // timestamp when was created
     timeEnded?: number; // timestamp when was finished
     paused?: boolean; // current state
-    isAutoPauseEnabled?: boolean; // auto pause after current round
+    isAutoStopEnabled?: boolean; // auto pause after current round
     interrupted?: boolean; // it was paused by force
```

Comment:
Now that coinjoin cannot be paused manually, `interrupted` always equals `paused`. You can remove the `interrupted` param.

Reply:
It is more precise, but I'd stick to whatever the UI is showing.

Reply:
02d04c49df70db4f476198875016885bab17a940